pkg/util/util.go

```diff
@@ -211,11 +212,15 @@ func BreakingChangeAllowed(left string, right string) (bool, error) {
 }
 
 func PrintDelimiterLine(delimiterChar string) {
```

Comment:
We may as well note this func is deprecated
```suggestion
// Deprecated, to be removed in v4
func PrintDelimiterLine(delimiterChar string) {
```

Reply:
I had thought about it and didn't do it. But I guess it makes sense.

Reply:
yep probably fine either way (I "approved" with this optional suggestion). Accept if you want to, but either way 🚀 🙂

Reply:
Fixed